site/composer.json

```diff
@@ -29,7 +29,8 @@
     "php-ds/php-ds": "1.2.0",
     "cboden/ratchet": "0.4.3",
     "textalk/websocket": "1.3.1",
-    "egulias/email-validator": "2.1.18"
+    "egulias/email-validator": "2.1.18",
+    "maennchen/zipstream-php": "^2.1"
```

Comment:
ah, sorry, within my comment, I did not include the pinned version. Please change this to be `2.1.0` instead of `^2.1`.